Source/DafnyCore/AST/Grammar/TokenNewIndentCollector.cs

```diff
@@ -352,10 +342,27 @@ private class Indentations {
           }
       }
     }
+  }
+
+  public void SetTypeIndentation(Type type) {
+    var tokens = type.OwnedTokens.ToList();
```

Comment:
Setting up type indentation was also not recursive post-resolution, so I had to change that.